lib/galaxy_test/api/test_histories.py

```diff
@@ -544,6 +544,12 @@ def test_anonymous_can_import_published(self):
             }
             self.dataset_populator.import_history(import_data)
 
+    def test_publish_non_alphanumeric(self):
+        history_name = "تاریخچه"
+        history_id = self.dataset_populator.new_history(name=history_name)
+        response = self.dataset_populator.make_public(history_id)
+        assert response["username_and_slug"]
```

Comment:
Can you assert the actual slug, and maybe we can use something not religious ?

Reply:
The `SharingStatus` model doesn't have the slug itself.
Should I split it from this one?

Reply:
like this:
```suggestion
        assert response["username_and_slug"].split("h/")[1] == history_name
```

Reply:
You can also assert that the history_name is in response["username_and_slug"], whatever you prefer